plugins/pmsmart/src/main/java/org/itsallcode/whiterabbit/plugin/pmsmart/web/page/DateSelector.java

```diff
@@ -23,9 +30,15 @@ public DateSelector(Driver driver, Element selector)
 
     public void select(LocalDate day)
     {
+        if (day.isEqual(CRITICAL_DATE))
+        {
+            LOG.debug("Selecting critical date");
```

Comment:
Please remove debug code

Reply:
OK, done.
Is it OK to add sample code to class Tester?

Reply:
Yes, because its useful for debugging. But the Tester should be self contained, i.e. have no dependencies to local files.